crates/server/src/admin/handlers/context.rs

```diff
@@ -34,12 +37,26 @@ pub async fn get_context_handler(
 
     match context {
         Ok(ctx) => match ctx {
-            Some(context) => ApiResponse {
-                payload: GetContextResponse {
-                    data: ContextObject { context },
-                },
+            Some(context) => {
+                let context_identities = state
+                    .ctx_manager
+                    .get_context_owned_identities(context.id)
+                    .map_err(|err| parse_api_error(err).into_response())
+                    .unwrap_or_default()
```

Comment:
looks to me like we're throwing this error away, yeah? we should have a `err.into_response` somewhere, unless you think it's irrelevant to share store-errors

Reply:
Idea is to not expose internal errors outside. So we map all errors to message we want to expose

Reply:
pub fn parse_api_error(err: eyre::Report) -> ApiError {
    match err.downcast::<ApiError>() {
        Ok(api_error) => api_error,
        Err(original_error) => ApiError {
            status_code: StatusCode::INTERNAL_SERVER_ERROR,
            message: original_error.to_string(),
        },
    }
}

Reply:
Do we really want to expose internal error outside? User can react only on invalid contextId. Not sure what else he can fix?

Reply:
You're probably right, if this fails, for a database issue, then maybe `get_context` failed as well and we would've returned.

Just concerned about saying we don't have context IDs when we do, but had an internal error.

There's always a chance retrying might work, maybe..

Reply:
Make sense. Updated to return generic error if there is some internal issue